content/docs/reference/metadata/v1alpha1/kubeflow-metadata-api-spec.html

```diff
@@ -0,0 +1,3811 @@
+<!DOCTYPE html>
```

Comment:
Add the following page metadata at the top of the file:

```
+++
title = "Metadata API Reference"
description = "Reference documentation for the Kubeflow Metadata API"
weight = 10
+++
```

The page metadata will prompt the website to include the navigation and styling around the API page, which makes it easier for people to find their way around the docs. See the [Pipelines API spec](https://github.com/kubeflow/website/blob/master/content/docs/pipelines/reference/api/kubeflow-pipeline-api-spec.html) as an example.

Reply:
Done